config/cesm/machines/config_machines.xml

```diff
@@ -358,11 +358,11 @@ This allows using a different mpirun command to launch unit tests
     <NODENAME_REGEX>casper*</NODENAME_REGEX>
     <OS>LINUX</OS>
     <COMPILERS>pgi,intel,nvhpc,pgi-gpu,nvhpc-gpu</COMPILERS>
-    <MPILIBS compiler="pgi" >openmpi,mpi-serial</MPILIBS>
-    <MPILIBS compiler="nvhpc" >openmpi,mpi-serial</MPILIBS>
-    <MPILIBS compiler="intel" >openmpi,mpi-serial</MPILIBS>
-    <MPILIBS compiler="pgi-gpu" >openmpi,mpi-serial</MPILIBS>
-    <MPILIBS compiler="nvhpc-gpu" >openmpi,mpi-serial</MPILIBS>
+    <MPILIBS compiler="pgi" >openmpi</MPILIBS>
```

Comment:
again - since all of the compilers use openmpi you don't need a separate entry for each compiler.
```
<MPILIBS>openmpi</MPILIBS>
```
is enough.

Reply:
Thanks, @jedwards4b . I revised the settings for mpilib based on your suggestion and the new test run worked fine.